packages/api-tests/test/window.core.spec.js

```diff
@@ -616,6 +616,31 @@ describe('WindowCore API', function(done) {
       return chainPromises(steps);
     });
 
+    it.only('Should wrap a native window in a containerjs window object #ssf.Window.wrap', function() {
```

Comment:
@BenLambertNcl Just noticed this `it.only` was left in, preventing other tests from running.